Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'nokogiri', '>= 1.6.7.1'
 
 gem 'pg'
 gem 'spree', github: 'openfoodfoundation/spree', branch: 'step-6a', ref: '5a76d45'
-gem 'spree_i18n', github: 'spree/spree_i18n', branch: '1-3-stable'
+gem 'spree_i18n', github: 'openfoodfoundation/spree_i18n', branch: '1-3-stable'
```

Comment:
I didn't realise that we weren't already using our own fork of `spree_i18n`. Is it not possible to just add the translations submitted in https://github.com/openfoodfoundation/spree_i18n/pull/1 to our own `en.yml`? I would prefer that option if possible, rather than forking all the things.

Reply:
OK I put the changes back to en.yml :)

Reply:
@sauloperez for info, please let us no if any objection, else we push again for Aus Review and hope it can be merged before next release so that we see better what translations are still missing ;-)

Reply:
Fine for me. Let's move forward :+1: